applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_adjoint_static_solver.py

```diff
@@ -96,12 +105,19 @@ def Initialize(self):
             self.response_function = StructuralMechanicsApplication.AdjointLinearStrainEnergyResponseFunction(self.main_model_part, self.settings["response_function_settings"])
         elif response_type == "adjoint_nodal_reaction":
             self.response_function = StructuralMechanicsApplication.AdjointNodalReactionResponseFunction(self.main_model_part, self.settings["response_function_settings"])
+        elif response_type == "adjoint_max_stress":
+            self.response_function = StructuralMechanicsApplication.AdjointMaxStressResponseFunction(self.main_model_part, self.settings["response_function_settings"])
+        elif response_type == "adjoint_aggregated_stress":
+            self.response_function = StructuralMechanicsApplication.AdjointAggregatedStressResponseFunction(self.main_model_part, self.settings["response_function_settings"])
         else:
             raise Exception("invalid response_type: " + response_type)
 
         self.sensitivity_builder = KratosMultiphysics.SensitivityBuilder(self.settings["sensitivity_settings"], self.main_model_part, self.response_function)
         self.sensitivity_builder.Initialize()
 
+        self.max_number_of_threads = KratosMultiphysics.OpenMPUtils.GetNumThreads()
+        KratosMultiphysics.OpenMPUtils.SetNumThreads(1)
```

Comment:
blocking until this is resolved
As you pointed out this should not go into Master